### DISCUSSION TITLE
Unable to use custom fonts

### DISCUSSION THREAD

**Discussion by obellish** [BinaryKits/ZPLUtility#166] [Q&A]
I use the BinaryKits.Viewer package for a custom barcoding tool we use, and have noticed there's no inherit way to show custom fonts loaded on the printer, is there a workaround for this?

**Comment by gayathrijosyula** [comment]
I have utilized the following approach to set my custom fonts. Please inform me if it proves effective. 

https://github.com/BinaryKits/BinaryKits.Zpl#font-mapping:~:text=Font%20mapping,)%3B%0A%7D

![image](https://github.com/BinaryKits/BinaryKits.Zpl/assets/121975085/5c81b445-42a5-412f-b537-32626d93ff48)
